tests/e2e/test_amq_streamer_creation.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.amq import AMQ
+from ocs_ci.framework.testlib import E2ETest, workloads
+
+from ocs_ci.framework.testlib import tier1, ManageTest
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+@pytest.fixture(scope='function')
+def test_fixture_amq(request,storageclass_factory):
+    global CEPHFS_SC_OBJ
+    CEPHFS_SC_OBJ = storageclass_factory(interface=constants.CEPHFILESYSTEM,sc_name='amq-workload')
```

Comment:
please refer pg_sql test script and have similar storage class factory here, you can set default name in yaml for that.  Also this will get rid of global.

Reply:
Removed global